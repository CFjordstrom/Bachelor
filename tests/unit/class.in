[a-ch-j0-9]