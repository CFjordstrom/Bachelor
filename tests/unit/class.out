-mindfa
{
#1 -> [0-9a-ch-j] #2;
#2 -> ();
}
#1